src/SFA.DAS.AssessorService.Application/Infrastructure/OuterApi/GetStandardByIdResponse.cs

```diff
@@ -25,7 +25,13 @@ public class StandardDetailResponse
         public int Level { get; set; }
 
         [JsonProperty("version")]
-        public decimal Version { get; set; }
+        public string Version { get; set; }
```

Comment:
This refers to multiple classes, where you have added a string Version and interger VersionMajor and VersionMinor, properties, shouldn't the string Version property be a calculated property based on VersionMajor and VersionMinor?

Reply:
Given the data from Ifate comes in as a string, it seemed best to store that as is so we hold the original value. The VersionMajor VersionMinor is the calculated aspect from splitting the original field. The sole purpose of the split was purely for ordering purposes at the database level. It is a little bit duplicated but it seemed the simplest option.

The same applies here in the assessors space, only using the major and minor for ordering where version order is important while maintaining the original string value. It is probably more appropriate here for a calculated value but I was trying to limit the changes given how much of an impact just changing the type caused. In addition I knew we were coming back into the assessors service post approvals which would allow further tidy up once back in the space.